framework/src/main/java/org/checkerframework/framework/stub/StubParser.java

```diff
@@ -715,11 +778,16 @@ private void processCallableDeclaration(CallableDeclaration<?> decl, ExecutableE
                 annotate(
                         methodType.getReceiverType(),
                         decl.getReceiverParameter().get().getAnnotations());
+                annotate(
```

Comment:
A comment here would help explain why two `annotate` calls for the receiver type are here.

Reply:
Done above.